navigation-compose/src/main/kotlin/com/freeletics/khonshu/navigation/compose/NavHost.kt

```diff
@@ -61,6 +61,66 @@ public fun NavHost(
     deepLinkPrefixes: Set<DeepLinkHandler.Prefix> = emptySet(),
     navController: NavHostController = rememberNavController(),
     destinationChangedCallback: ((BaseRoute) -> Unit)? = null,
+) {
+    InternalNavHost(
```

Comment:
We can annotate two functions with `@NonRestartableComposable`, just an optimization.

Reply:
👍  will open a pr for this